### ISSUE TITLE
Combobox does not have focus indicator

### ISSUE THREAD

**Issue by heddendorp** [ng-primitives/ng-primitives#288]
### Affected Primitive

Combobox

### Version

0.50.0

### Bug Description

Currently it seems impossible to style the combobox for focus, only the input seems to receive a focus attribute. Even for the input it seems to be not documented. Additionally it would be nice to explain the combobox specific css variables in the docs.

**Comment by ashley-hunter** [comment]
Thank you for the suggestion, yes I should certainly add that. You can style it using the :focus-within pseudo selector, but I will add the focus data attribute, good idea!
